src/scanning/scan.ts

```diff
@@ -17,9 +17,8 @@ import {ParsedDocument} from '../parser/document';
 
 import {Scanner} from './scanner';
 
-export async function scan(
-    document: ParsedDocument<any, any>,
-    scanners: Scanner<any, any, any>[]): Promise<ScannedFeature[]> {
+export async function scan<A, V, P extends ParsedDocument<A, V>>(
```

Comment:
They're just passed in through to ParsedDocument. I improved the docs on ParsedDocument.